Cargo.toml

```diff
@@ -19,10 +19,9 @@ surf = "2.2"
 tiny_http = "0.6"
 tokio = { version = "1", features = ["full"] }
 toml = "0.4.2"
-twitch_api2 = { version = "0.5.0-alpha", features = ["client", "helix", "surf_client"], path="twitch_api2/" }
+twitch_api2 = { version = "0.5.0-alpha", features = ["client", "helix", "surf_client", "twitch_oauth2"], git = "https://github.com/Emilgardis/twitch_api2/"  }
```

Comment:
`version` does nothing here. Instead a git tag/branch/commit should be specified

Reply:
done